compiler/src/Language/Bond/Codegen/Java/SerializationMethods.hs

```diff
@@ -278,3 +280,13 @@ readMap java keyType valueType fieldName depth =
         countLocal = "count" ++ show depth
         keyLocal = "k" ++ show depth
         valueLocal = "v" ++ show depth
+
+readBonded :: MappingContext -> Type -> String -> Text
+readBonded java bondedType fieldName =
```

Comment:
`bondedType` refers to the type inside the bonded. This is used as `BT_Bonded t -> readBonded java t varName`.

If you found the name misleading, I'm happy to change it.